### ISSUE TITLE
Should the license be BSD rather than MIT?

### ISSUE THREAD

**Issue by pombredanne** [moodmosaic/Fare#62]
Since this project is a port of https://www.brics.dk/automaton/ which is BSD-licensed, I wonder if this project should not use the same BSD license rather than MIT?

**Comment by moodmosaic** [comment]
Yes, that's a good point, I think. Feel free to open a PR for it. Thank you 👍

**Comment by garcipat** [comment]
if you change that, can you also fix the License Url?
its currenlty pointing to https://github.com/moodmosaic/Fare/blob/master/LICENCE instead of https://raw.githubusercontent.com/moodmosaic/Fare/master/LICENSE

**Comment by moodmosaic** [comment]
https://www.nuget.org/packages/Fare/ has a License that points to https://github.com/moodmosaic/Fare/blob/master/LICENSE

**Comment by garcipat** [comment]
But thats wrong link. this leads to the github file site like you open it to read manually in the repo but not with the raw content. If you want to download the content of this link you get a html page and for complying with those licenses it is difficult to get the real text out of that website.
This is problematic and 99% of all other projects discovered that and link to the raw file.

**Comment by garcipat** [comment]
Thats completly fine. Just wanted to mention it so it can be fixed for the next release. Thank you <3

**Comment by moodmosaic** [comment]
A new release will be out soon. I will try to tackle this there. Thanks!
